script.js

```diff
@@ -408,6 +442,28 @@ function handleData([
  * HANDLE ADDING MAP LAYERS
  *****************************************/
 
+// Styles
+const scoreFillColors = {
```

Comment:
just a note to make sure this doesn't conflict with @jimboweb's method of using an array

Reply:
@river-honer you can remove this code block.